### PR TITLE
Minor test improvements

### PR DESCRIPTION
**Pull request by pdhamdhere**
1. Add timestamps to logs
2. Reduce volume size 
3. Reduce number of iterations for concurrency tests

Saves approximately 8-9minutes each CI run.

### CODE REVIEW

File: vmdk_plugin/sanity_test.go [hunk 1]
```diff
@@ -250,6 +253,9 @@ func TestSanity(t *testing.T) {
 				volumeName, elem.endPoint)
 		}
 	}
+	fmt.Printf("%s END: Running TestSanity on  %s (may take a while)...\n",
```
Comment (shuklanirdesh82) [inline vmdk_plugin/sanity_test.go]: minor: better to stick with other logging in this file using t.Logf() ... e.g. ```line 252```

**Comment by shuklanirdesh82** [comment]
> Saves approximately 8-9minutes each CI run.

Great! Thanks for helping here!

File: vmdk_plugin/sanity_test.go (hunk 1 above)
Comment (pdhamdhere) [inline vmdk_plugin/sanity_test.go]: True but I am trying to be consistent with START block at L213 'fmt.Printf("%s START:'

Comment (shuklanirdesh82) [inline vmdk_plugin/sanity_test.go]: I see ... fine with me. We will take care during massive cleanup.

**Comment by pdhamdhere** [comment]
Thanks @shuklanirdesh82 for review. Since this is test-only minor change, merging with 1 approval.

**Comment by pdhamdhere** [comment]
#1057

**Pull request merged**